var/spack/repos/builtin/packages/rocm-debug-agent/package.py

```diff
@@ -15,15 +15,18 @@ class RocmDebugAgent(CMakePackage):
 
     maintainers = ['srekolam', 'arjun-raj-kuppala']
 
+    version('3.7.0', sha256='d0f442a2b224a734b0080c906f0fc3066a698e5cde9ff97ffeb485b36d2caba1')
     version('3.5.0', sha256='203ccb18d2ac508aae40bf364923f67375a08798b20057e574a0c5be8039f133')
 
     variant('build_type', default='Release', values=("Release", "Debug"), description='CMake build type')
 
     depends_on('cmake@3:', type='build')
-    for ver in ['3.5.0']:
+
+    for ver in ['3.5.0', '3.7.0']:
         depends_on('hsa-rocr-dev@' + ver, type='link', when='@' + ver)
         depends_on('hsakmt-roct@' + ver, type='link', when='@' + ver)
-        depends_on("elfutils", type='link', when='@' + ver)
+
```

Comment:
depends_on('rocm-dbgapi@3.7.0', type='link', when='@3.7.0')

Reply:
see CMake Error at test/CMakeLists.txt:40 (find_package):
     20      No "FindHIP.cmake" found in CMAKE_MODULE_PATH.

Reply:
Hm, I didn't test this one, you're right. Adding hip to `depends_on` does not resolve it.

Reply:
Ok, that's because they use `find_package(HIP REQUIRED MODULE)`, I've just added CMAKE_MODULE_PATH variable instead.